src/data-workspace/status-tag/status-tag.js

```diff
@@ -0,0 +1,12 @@
+import i18n from '@dhis2/d2-i18n'
+import { Tag } from '@dhis2/ui'
+import React from 'react'
+import { IconApprovalReady } from './icon-approval-ready.js'
+
+const StatusTag = () => (
+    <Tag neutral icon={<IconApprovalReady size={12} />}>
+        {i18n.t('Ready for approval')}
```

Comment:
I could not find a way to get the current status of a workflow. Are we waiting on an API change or is there a separate endpoint we are meant to query?